addons/website/static/src/scss/website.scss

```diff
@@ -176,6 +176,16 @@ $-seen-urls: ();
     }
 }
 
+#oe_main_menu_navbar:not(.o_hidden) + #wrapwrap {
+    .o_header_affixed, .modal {
+        // For these elements, in #wrapwrap, position top should be size of the navbar
+        top: $o-navbar-height;
+    }
+    .modal {
+        height: calc(100vh - #{$o-navbar-height});
```

Comment:
Won't `height: auto` + `bottom: 0` have the same effect? I wonder why this is not like that in bootstrap in the first place? Maybe we should do that directly in bootstrap_review.scss?

Reply:
https://github.com/twbs/bootstrap/pull/27592/commits/9c6ab5362dade181b5cf8393eb7c5176aa557bdd
Boostrap removed `bottom: 0` and added `height: 100%`  in V4.2 because of a bug on Android...
So i guess it's better to leave it like that. what do you think ?

Reply:
Ok, add a comment a comment in the code ;)

Reply:
ok done